### PR TITLE
added README.md file

### PR DESCRIPTION
**Pull request by ChrisEdS**
README.md with some hints on how to use Theming in general.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## theme-example for ownCloud
+
+### Use Theme
+To use the theme clone or download this repository to the `owncloud\apps' folder on your server. Please note that the Apache user needs access rights to the folder and its content. To adjust the permissions use `chmod -R www-data:www-data theme-example`.
```
Comment (phil-davis) [inline README.md]:
```suggestion
To use the theme clone or download this repository to the `owncloud\apps` folder on your server. Please note that the Apache user needs access rights to the folder and its content. To adjust the permissions use `chmod -R www-data:www-data theme-example`.
```

**Review by phil-davis — COMMENTED** [review]
Words look OK. I am not totally familiar with all the technical detail.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+## theme-example for ownCloud
+
+### Use Theme
+To use the theme clone or download this repository to the `owncloud\apps` folder on your server. Please note that the Apache user needs access rights to the folder and its content. To adjust the permissions use `chmod -R www-data:www-data theme-example`.
```
Comment (mmattel) [inline README.md]: Proposal:
`### Use Theme` --> `### Use the theme-example App` 
`To use the theme clone...` --> `To use the theme-example app, clone...`
`To adjust the permissions...` --> `As an example, use following command to adjust the permissions...`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+To edit images it is best to open the original images from the 'core\img' folder with Gimp or Photoshop. After you have made changes, overwrite the originals.
+
+### Change values
+Please have a look at the file `defaults.php` if you want to change texts. The copyright text on the start page and other texts are well documented there.
+
+### Change design values
+In the file `styles.css` you will find the adjustments we have preconfigured for you. You can change everything you want there. More classes and their values can be found easily with the browser console (F12 key). There you can also test directly in the browser how the change of values will be reflected. If it fits you can transfer the values to styles.css.
+
+### Integrity check
+Since files in the theme must be changeable, it has not been signed by us. Therefore an 'integrity check warning' appears in your ownCloud. You can deactivate it specifically for the theme:
```
Comment (mmattel) [inline README.md]: `appears in your ownCloud` --> `appears in your ownCloud instance`
`You can deactivate it specifically for the theme:` --> `You can deactivate it for the theme-example app:`

Comment (ChrisEdS) [reply README.md]: @mmattel Thanks Martin, I'll take your suggestions.

File: README.md (hunk 2 above)
Comment (ChrisEdS) [reply README.md]: @mmattel Thanks Martin, I'll take your suggestions.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+## theme-example for ownCloud
+
+### Use the theme-example App
+To use the theme-example app, clone or download this repository to the `owncloud\apps' folder on your server. Please note that the Apache user needs access rights to the folder and its content. As an example, use following command to adjust the permissions: `chmod -R www-data:www-data theme-example`.
```
Comment (mmattel) [inline README.md]: There is a wrong apostroph instead a backtick at the end of owncloud\apps.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+If you change the name of the app from `theme-example` to another name, be sure to change the name in the file `appinfo\info.xml` as well. The folder name and the `<id>' must be identical.
```
Comment (mmattel) [inline README.md]: Same here at `<id>`

File: README.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+### Edit images
+To edit images it is best to open the original images from the 'core\img' folder with Gimp or Photoshop. After you have made changes, overwrite the originals.
```
Comment (mmattel) [inline README.md]: Pls switch to backticks at core\img

File: README.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+### Change values
+Please have a look at the file `defaults.php` if you want to change texts. The copyright text on the start page and other texts are well documented there.
+
+### Change design values
+In the file `core/css/styles.css` you will find the adjustments we have preconfigured for you. You can change everything you want there. More classes and their values can be found easily with the browser console (F12 key). There you can also test directly in the browser how the change of values will be reflected. If it fits you can transfer the values to styles.css.
+
+### Integrity check
+Since files in the theme must be changeable, it has not been signed by us. Therefore an 'integrity check warning' appears in your ownCloud instance. You can deactivate it for the theme-example app:
```
Comment (mmattel) [inline README.md]: Same here at integrity check warning.
+
`theme-example app:` —> `theme-example app by adding the following config parameter to your config.php file:`

**Comment by mmattel** [comment]
Looks good. Some additional things found that need attention.

File: README.md (hunk 7 above)
Comment (ChrisEdS) [reply README.md]: Good catch! Thanks.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
+## theme-example for ownCloud
+
+### Use the theme-example App
+To use the theme-example app, clone or download this repository to the `owncloud\apps` folder on your server. Please note that the Apache user needs access rights to the folder and its content. As an example, use following command to adjust the permissions: `chmod -R www-data:www-data theme-example`.
+
+If you change the name of the app from `theme-example` to another name, be sure to change the name in the file `appinfo\info.xml` as well. The folder name and the `<id>` must be identical.
+
+### Edit images
+To edit images it is best to open the original images from the 'core\img` folder with Gimp or Photoshop. After you have made changes, overwrite the originals.
```
Comment (mmattel) [inline README.md]: The final correction: you have to change the leading apostroph to backtick in core\img.
For highlighting, you need to have a leading and trailing backtick. Else the render enginge confuses.
This must have been slipped thru...

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]
Please squash before merging - THX

**Comment by mmattel** [comment]
After squashing, you still have the leading apostroph instead a backtick in core\img.
Pls see my comment in https://github.com/owncloud/theme-example/pull/8#discussion_r375128509

File: README.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
+## theme-example for ownCloud
+
+### Use the theme-example App
+
+To use the `theme-example` app, clone or download this repository to the `owncloud\apps` folder on your server. Please note that the Apache user needs access rights to the folder and its content. As an example, use the following command to adjust the permissions: `chmod -R www-data:www-data theme-example`.
+
+If you change the name of the app (the folder name) from `theme-example` to another name, be sure to change the `<id>` in the file `appinfo\info.xml` as well. The folder name and the `<id>` must be identical.
+
+### Edit images
+To edit images it is best to open the original images from the `core\img` folder with Gimp or Photoshop. After you have made changes, overwrite the originals.
```
Comment (ChrisEdS) [inline README.md:10]: @mmattel Now it should fit. :)

**Comment by mmattel** [comment]
@ChrisEdS the address written `https://central.owncloud.com` needs to be a `.org` address or has there something changed / is .com a new and upcomeing one?

**Comment by ChrisEdS** [comment]
> @ChrisEdS the address written `https://central.owncloud.com` needs to be a `.org` address or has there something changed / is .com a new and upcomeing one?

I already changed this with my last commit.

File: README.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+Since files in the theme must be changeable, it has not been signed by us. Therefore an `integrity check warning` appears in your ownCloud instance. You can deactivate it for the `theme-example` app:
+
+```
+'integrity.ignore.missing.app.signature' => [
+      theme-example,
+ ],
+```
+
+### Questions
+If you have any questions you can ask them on https://central.owncloud.com.
```
Comment (phil-davis) [inline README.md]:
```suggestion
If you have any questions you can ask them on https://central.owncloud.org.
```
This should be org ?

**Comment by mmattel** [comment]
> I already changed this with my last commit.

nope, sorry, it´s not in. Still .com

**Comment by ChrisEdS** [comment]
> > I already changed this with my last commit.
> 
> nope, sorry, it´s not in. Still .com

Damn. Force pushed again.

**Review by mmattel — APPROVED** [review]

**Comment by mmattel** [comment]
Great 👍 merge it !

**Pull request merged**